sqlmesh/core/renderer.py

```diff
@@ -106,6 +106,7 @@ def _render(
                     cache_key[0] if not self._only_execution_time else None,
                     cache_key[1] if not self._only_execution_time else None,
                 ),
+                "default_catalog": self._default_catalog,
```

Comment:
can you refactor this a bit. add default_catalog to kwargs, and add kwargs to the cache key on line 98?

Reply:
yeah that was moreso just putting some pen to the paper so I have a starting point, I am not sure if there is a single source of truth "canonical" place you add variables to the render context. I also think, based on the CI error, that it might already be available to the context 🤔 

I have a feeling this might be more of a testing/documentation exercise on the sqlmesh side.

Reply:
So I validated that default catalog did not work and was thus not in the context. But its looks like I found the right place to add it. This should help users who need to run `GRANTs` or any other command which parses the body as a literal since I THINK at least macros will be expanded.